lab-cameron/test/move-routes-test.js

```diff
@@ -0,0 +1,206 @@
+          done();
+        });
+      });
+      it('Should respond with status 400 on a bad request', done => {
+        chai.request(server)
+        .post('/api/move')
+        .send()
+        .end((err, res) => {
+          expect(res).to.have.status(400);
+          done();
+        });
+      });
+    });
+  });
+
+  describe('#GET', function() {
+    let moves = [];
+    before(done => {
+      chai.request(server)
+      .post('/api/move')
+      .send({name: 'hyper beam', attack: 100, power: 5})
+      .end((err, res) => {
+        let move = JSON.parse(res.text.toString());
+        moves.push(move);
+        done();
+      });
+    });
+    after(done => {
+      moves.forEach(ele => {
+        chai.request(server)
+        .delete(`/api/move/${ele._id}`)
+        .end();
+      });
+      done();
+    });
+    describe('requests made to /api/move/:id', function() {
+      it('Should respond with status 200 on a proper request', done => {
+        let testmove = moves[0];
+        chai.request(server)
+        .get(`/api/move/${testmove._id}`)
+        .send()
+        .end((err, res) => {
+          expect(res).to.have.status(200);
+          done();
+        });
+      });
+      it('Should respond with status 400 on a bad request', done => {
+        chai.request(server)
+        .get(`/api/move/1234`)
+        .send()
+        .end((err, res) => {
+          expect(res).to.have.status(400);
+          done();
+        });
+      });
+      describe('requests made to /api/move', function() {
+        it('Should respond with status 200 on a proper request and return whole schema', done => {
+          chai.request(server)
+          .get('/api/move')
+          .send()
+          .end((err, res) => {
+            expect(res).to.have.status(200);
+            done();
+          });
+        });
+        it('Should respond with status 400 on a bad request', done => {
+          chai.request(server)
+          .get(`/api/move/1234`)
+          .send()
+          .end((err, res) => {
+            expect(res).to.have.status(400);
+            done();
+          });
+        });
+      });
+    });
+  });
+
+  describe('#PUT', function() {
+    let moves = [];
+    before(done => {
+      chai.request(server)
+      .post('/api/move')
+      .send({name: 'hyper beam', attack: 100, power: 5})
+      .end((err, res) => {
+        let move = JSON.parse(res.text.toString());
+        moves.push(move);
+        done();
+      });
+    });
+    after(done => {
+      moves.forEach(ele => {
+        chai.request(server)
+        .delete(`/api/move/${ele._id}`)
+        .end();
+      });
+      done();
+    });
+    describe('requests made to /api/move/:id', function() {
+      it('Should respond with status 200', done => {
+        let testmove = moves[0];
+        chai.request(server)
+        .put(`/api/move/${testmove._id}`)
+        .send({name: 'psybeam'})
+        .end((err, res) => {
+          expect(res).to.have.status(200);
```

Comment:
You should also have an additional ``expect`` statement testing to make sure your name is equal to ``psybeam`` to confirm the change happened.